include/picongpu/fields/incidentField/Solver.hpp

```diff
@@ -188,7 +188,10 @@ namespace picongpu
                     typename T_CurlIncidentField,
                     typename T_FunctorIncidentField,
                     uint32_t T_axis>
-                inline void updateField(Parameters<T_axis> const& parameters, float_X const curlCoefficient)
+                inline void updateField(
+                    Parameters<T_axis> const& parameters,
+                    float_X const curlCoefficient,
+                    bool transversalContiguousHuygensSurface)

```

Comment:
doxygen string missing

Reply:
great that you review your own code @psychocoderHPC 😉 
Could you add the doxygen string?

Reply:
I assume not needed